EiDiA_backend/src/models/document.js

```diff
@@ -45,23 +50,25 @@ const DocumentSchema = new mongoose.Schema({
         type: String,
         required: true,
     },
-    attributes: {
-        type: [
-            {
-                attributeId: {
-                    type: mongoose.Schema.Types.ObjectId,
-                    required: true,
-                    index: true,
-                }
-            },
-            {
-                value: {
-                    required: true,
-                    index: true,
-                }
-            },
-        ]
+    comment: {
+        type: String,
+        index: true,
     },
+    attributes: [
```

Comment:
In meinem Branch habe ich das Schema so definiert:


```suggestion
    attributes: [
        {
            attributeId: {
                type: mongoose.Schema.Types.ObjectId,
                required: true,
                index: true,
            },
            value: {}
        },
    ],
});
```
Das Schema ist passend für POST-Requests, die so definiert sind:
`{
    "attributeData": [{"attributeId": "5ef9e3d2c4664e04e4003999", "value": "testText1"}, {"attributeId": "5ef9e3d2c4664e04e4003999", "value": "testText2"}]
}`

So war das doch eigentlich gedacht, oder?

Reply:
Siehe #54